<h1>Aluminio Catalo-combustivel</h1>

A reação quimica:

2Al + 6H2O + NaOH -> 2Al(OH)3 + 3H2 + NaOH 

Produz hidrogenio gasoso.

Considerando:

- A proporção ideal de queima do H -> 29% H + 71% Ar
- Um queimador de 100cm3

Teriamos então 29cm3 de H2.

Considerando:

- O calor de combustao do H -> 121MJ/kg
- O volume do H2 gasoso -> 22,4L / mol
- O peso molar do H2 de aprox 2g

Temos que cada cm3 conteria então 2g / 22400 = 8.929 E-5 g de H. Vezes 29 = 2.589mg de H no queimador de 100cm3 com 29% de H.

O resultado da combustão de 2.589mg de H seria 313J.

Considerando:

- O peso molar do Aluminio de aprox 27g/mol

Teriamos que para cada cm3 de H produzido pela reação teriamos 8.036 E-4 g de Al convertido em sal. Vezes 29 = 23,3mg

Considerando:

- O peso molar da agua de aprox 18g/mol

Teriamos que para cada cm3 de H produzido pela reação teriamos 1.6mg de agua removida. Vezes 29 = 46,6mg.

Para conseguir então 1J da reação temos que 7.445 E-5 Al e 1.489 E-4 H20 são consumidos.

Considerando um cenario que exigiria 4W constantes por um mês teriamos a necessidade de 10M368 J.

Essa reação consumiria entao 772,93g de Al e 1543.7952ml de agua. No entanto, podemos considerar que a queima do Hidrogenio produz Agua. Se for possivel recolher essa agua formada, poderiamos ter um consumo de agua menor.

Considerando:

- Que o queimador de 100cm3 tenha lados com o mesmo comprimento

Temos 46.416mm de lado. 

Considerando:

- Uma folha de papel aluminio de 45mm de lado e 0.016mm de espessura
- A densidade do Al de aprox 2.7g/cm3

Temos que para cada g de aluminio teriamos 5.144cm dessa fita. Para nosso cenário de 772.93g teriamos 39.76m de fita.

Calculando pelo volume de Al, um cilindro dessa folha enrolada em outro cilindo de 1cm de diam teria:

$\pi . r^2 . 4 - \pi . 0.5^2 . 4 = 772.93 / 2.7$

4.8cm de raio 

Essa folha de aluminio teria 1.7892m2. Sendo o custo de folha de Al 1.6 Reais por m2, teriamos o custo de 2.87 Reais por mes de funcionamento.

Também é um cenário de 2.88KW/h, com custo aproximado em SP de 2.49 Reais (ou seja, o custo de usar Al é quase igual ao de deixar na tomada).

O custo equivalente de queimar gasolina para um automovel é de 1.29 Reais.

Comprando-se Al de sucata por 1500 Reais a tonelada e fazendo pó de aluminio em vez de folhas, poderiamos ter um custo semelhante ao de queimar gasolina, desde que o custo de fazer o pó nao seja alto.

<h1>Experimento 1</h1>
<h2>Gerando hidrogenio</h2>
<h2>Arranjo Experimental</h2>
em 450ml de solução 3M de Hidróxido de sódio duas folhas de aluminio de 7.5cm x 15cm, com 40 micra de espessura foram acrescentadas. O sistema foi fechado hermeticamente e o gás produzido só podia sair por um tubo terminado numa agulha de 0.7mm de diametro de furo. O gás emitido pode ser inflamado com uma chama.<br/>

Abaixo o resultado da coleta de dados no experimento. A reação durou 51 minutos.

In [6]:
import numpy as np
import pandas


df = pandas.DataFrame({
   'hora': ['16h42', '16h43', '16h45', '17h03', '17h10', '17h11', '17h12', '17h20', '17h30', '17h34'],
   'tipo': ['inicio', 'gas', 'medida', 'medida', 'medida', 'medida', 'medida', 'medida', 'medida', 'termino'],
   'volume_cm3': [-1, -1, 57, 60, 60, 90, 90, 90, 90, -1],
   'tempo_s': [ -1, -1, 150, 100, 60, 68, 72, 71, 64, -1]
})

pandas.set_option( 'display.max_columns', None )
pandas.set_option( 'display.max_rows', None )

df

hora     tipo  volume_cm3  tempo_s
0  16h42   inicio          -1       -1
1  16h43      gas          -1       -1
2  16h45   medida          57      150
3  17h03   medida          60      100
4  17h10   medida          60       60
5  17h11   medida          90       68
6  17h12   medida          90       72
7  17h20   medida          90       71
8  17h30   medida          90       64
9  17h34  termino          -1       -1

In [8]:
df['vazao'] = df['volume_cm3'] / df['tempo_s']

df

hora     tipo  volume_cm3  tempo_s     vazao
0  16h42   inicio          -1       -1  1.000000
1  16h43      gas          -1       -1  1.000000
2  16h45   medida          57      150  0.380000
3  17h03   medida          60      100  0.600000
4  17h10   medida          60       60  1.000000
5  17h11   medida          90       68  1.323529
6  17h12   medida          90       72  1.250000
7  17h20   medida          90       71  1.267606
8  17h30   medida          90       64  1.406250
9  17h34  termino          -1       -1  1.000000

Pelos resultados podemos ver q durante o tempo de produção máxima (em algum ponto depois de 20 mins e antes de 27 mins) temos em média a produção de 1.25 cm3 de H.<br/>
<br/>
Tivemos então um perído de 20 mins com produção total de 1.5 litros de H.<br/>
<br/>
Podemos estimar uma produção nos primeiros 27 mins como uma progressão que passa por 0.38 apos 2 minutos e 0.6 após 20 minutos. Aproximada numa reta teriamos 2.037 e-4 x + 0.36. Integrando pelos 20 mins temos 545.4672 cm3 ou cerca de mais meio litro.<br/>
<br/>
Temos dois litros produzidos por uma massa de 2.43g de aluminio em 50 mins.<br/>
<br/>
Se utilizarmos a conta 2 mols de Al produzem 3 mols de H, teriamos com 2.43g de aluminio (0,09 mols) 0,135 mols de H, que seriam 0,27g, ou cerca de 3 litros. O experimento então produziu menos q isso, praticamente 1 mol de Al = 1 mol de H.<br/>
<br/>
A produção de H seguiu uma velocidade de reação provavelmente proporcional à area exposta, e o aumento do ritmo da reação pode talvez ser explicado pelo aumento de area causado pela corrosão irregular do Aluminio. Ela inicia com 0.36cm3 por 450cm2 de area de reação e aumenta por no min 20 até no max 27 mins até estabilizar em 1.25cm3 por 450cm2.<br/>
<br/>
Considerando o calor de combustão, temos 13.5 J sendo produzidos continuamente durante 20 a 30 mins da estabilidade da reação.<br/>
